docs/manage/_content.rst

```diff
@@ -106,26 +113,35 @@ If the computer where Kolibri server is running does not have access to Internet
 
 .. note::
   You must have an external drive (USB stick or hard disk) attached to your device.
 
 #. Click **Export** button in **Content** page.
 #. Select the local drive (export destination) where you wish to export **Kolibri** content, and click **Continue**.
-#. In the *Export to <name-of-your-drive>* page you will be able to see all the available content channels on your device. 
-  
+#. In the *Export to <name-of-your-drive>* page you will be able to see all the available content channels on your device.
+
   	.. image:: img/export-to.png
 	  :alt: Select from which channel you want to export to local drive.
 
 #. Click **Select** button for the desired channel, and wait for Kolibri to display the channel information and the topic tree.
 #. In the **Select content** page you will see all the details of the selected channel: description, version, total size and number of learning resources.
-#. Under **Choose content to export** you can browse the channel topics and individual resources. Use the *Select all* checkbox to import the content channel in full, or select only certain topics or resources. As you keep selecting, you will see the total number and size on disk under *Resources selected:*, and the remaining space on the destination drive.  
-#. Click **Export** button once you finish selecting all the desired content. 
+#. Under **Choose content to export** you can browse the channel topics and individual resources. Use the *Select all* checkbox to import the content channel in full, or select only certain topics or resources. As you keep selecting, you will see the total number and size on disk under *Resources selected:*, and the remaining space on the destination drive.
+#. Click **Export** button once you finish selecting all the desired content.
 #. Wait for Kolibri to export the selected content and click **Close**.
-#. Once the export is finished, safely disconnect the drive according to the recommended procedure for your operating system, and proceed to import channels on other devices. 
+#. Once the export is finished, safely disconnect the drive according to the recommended procedure for your operating system, and proceed to import channels on other devices.
 
 .. note:: This procedure makes a copy of the ``content`` folder located inside the ``.kolibri`` folder on your hard disk, and places it the ``KOLIBRI_DATA`` folder on the selected local drive. This structure is recognized by the **Import from local drive** command.
 
 	.. image:: img/kolibri-data-osx.png
-	  :alt: structure of the local drive folders with exported content channels
\ No newline at end of file
+	  :alt: structure of the local drive folders with exported content channels
+
+
+Peer-to-Peer Content Syncronization
+-----------------------------------
+
+``Coming soon!``
+
+
+.. Not yet. Peer to peer sync is a priority for Kolibri in the near future, but is not available yet.
```

Comment:
Maybe we should leave this out? "Coming soon" might get peoples' hopes up and we don't have a concrete timeline yet.

Reply:
Your beef is with @jeepurs :stuck_out_tongue:

Reply:
It's a comms issue.  @carinediaz @laurenlichtman how would you suggest to mitigate expectations? will 'Coming soon' up for many months be harmful?

Reply:
How about "stay tuned"?

Reply:
Changed!